packages/vuetify/src/components/VWindow/VWindowItem.ts

```diff
@@ -56,6 +57,9 @@ export default mixins<options & ExtractVue<[typeof Bootable]>>(
   },
 
   computed: {
+    classes (): object {
+      return this.groupClasses
```

Comment:
what's the point of computed here?

Reply:
Helps with typings. The `v-item-group` classes doesn't make it through.